classes/class-wc-klarna-order-management-scheduled-actions.php

```diff
@@ -13,48 +13,47 @@
 }
 
 /**
- * WC_Klarna_Order_Actions_Display class.
+ * WC_Klarna_Order_Management_Display_Actions class.
  *
  * Displays scheduled actions related to the order.
  */
-class WC_Klarna_Order_Actions_Display {
+class WC_Klarna_Order_Management_Display_Actions {
```

Comment:
This still does not match the name of the file. The file is named `class-wc-klarna-order-management-scheduled-actions`. So following the coding standards this should be named `WC_Klarna_Order_Management_Scheduled_Actions`.